web/src/main/java/com/epam/esm/controller/CertificateRESTController.java

```diff
@@ -0,0 +1,139 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.entity.Certificate;
+import com.epam.esm.mapper.impl.certificateMapper.CreateCertificateFromOnlyCertificateMapper;
+import com.epam.esm.model.certificate.ModelCertificate;
+import com.epam.esm.model.certificate.OnlyCertificate;
+import com.epam.esm.pagination.Pagination;
+import com.epam.esm.servise.impl.CertificateServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.hateoas.CollectionModel;
+import org.springframework.hateoas.Link;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please, avoid imports with *.
This type of import imports all classes in specified package (org.springframework.web.bind.annotation).
That's not really good, because in java we sometimes faced up with classes that have the same names and placed in different libraries/packages.
To avoid mistyping it's better to imports without *

Reply:
For removing authomatic convertion single imports to imports with *, you may configure your IDE. See 
![изображение](https://user-images.githubusercontent.com/41171038/194875177-db9c0351-34bb-4c1c-8f83-e11faaf3d2a7.png)

Reply:
Thanks, set it up. Long wanted, but forgot. ))